roles/nginxplus/files/conf/http/cdh_prod_geniza.conf

```diff
@@ -65,7 +65,7 @@ server {
         health_check interval=10 fails=3 passes=2;
     }
     location /indexcards {
-        return 301 https://genizaprojects.princeton.edu/indexcards$request_uri;
+        return 301 https://genizaprojects.princeton.edu/$request_uri;
```

Comment:
the pattern in the docs is https://genizaprojects.princeton.edu$request_uri; (no slash)